### PR TITLE
Add undocument properties to docs

### PR DESCRIPTION
**Pull request by harryadel**
Should resolve https://github.com/meteor/meteor/issues/12318

### CODE REVIEW

**Review by StorytellerCZ — APPROVED** [review]

File: packages/meteor/test_environment.js [hunk 1]
```diff
@@ -10,8 +12,29 @@ var testDriverPackageName = TEST_METADATA.driverPackage;
 
 // Note that if we are in test-packages mode neither of these will be set,
 // but we will have a test driver package
+/** 
+ *@memberof Meteor
+ * @summary Boolean variable.  True if running in test environment.
```
Comment (zodern) [inline packages/meteor/test_environment.js:17]: Sometimes this is false in test environments: https://github.com/meteor/meteor/blob/77df795928df55d1e114cb4d2a9143e433a9e773/tools/cli/commands.js#L1995-L1996

File: packages/meteor/meteor.d.ts [hunk 2]
```diff
@@ -486,7 +486,10 @@ export namespace Meteor {
   /** Global props **/
   /** True if running in development environment. */
   var isDevelopment: boolean;
+  var isModern: boolean;
+  var gitCommitHash: string;
```
Comment (zodern) [inline packages/meteor/meteor.d.ts:490]: `gitCommitHash` is undefined if the app isn't in a git repository.

File: packages/meteor/test_environment.js (hunk 1 above)
Comment (harryadel) [reply packages/meteor/test_environment.js:17]: What would be the proper description then? ` True if running in test environment, but can be set to false if test are being run in full-app mode`?

Comment (zodern) [reply packages/meteor/test_environment.js:17]: Maybe something like `True when running tests, except for when running in full-app mode`. Or maybe `True when running unit tests (false if running tests in full app mode)`.

**Comment by Grubba27** [comment]
Thanks again for the efforts @harryadel

**Pull request merged**